### DISCUSSION TITLE
Watching Multiple Namespace with KEDA Operator

### DISCUSSION THREAD

**Discussion by sidharthkumarpradhan** [kedacore/keda#4820] [Q&A / Need Help] [answered]
Want to watch 4 namespaces . using the below deploy

# Including admission webhooks
kubectl apply --server-side -f https://github.com/kedacore/keda/releases/download/v2.11.0/keda-2.11.0.yaml

In the Keda operator deployment file, tried to specify the namespaces in ',' a separate way, but not working

![image](https://github.com/kedacore/keda/assets/66263561/c2489fc4-c0fd-4ecb-9653-3ab49690024e)

Need help. Appreciate your time. Thanks

**Comment by sidharthkumarpradhan** [comment]
![image](https://github.com/kedacore/keda/assets/66263561/470c3519-a937-4f91-af49-b7b2247d385c)

With array also it is not working

**Comment by tomkerkhove** [comment, marked as answer]
Duplicate of https://github.com/kedacore/keda/discussions/1761
